app/code/Magento/GraphQl/etc/schema.graphqls

```diff
@@ -10,6 +10,7 @@ type Mutation {
 input FilterTypeInput @doc(description: "FilterTypeInput specifies which action will be performed in a query ") {
     eq: String @doc(description: "Equals")
     finset: [String] @doc(description: "Find in set. The value can contain a set of comma-separated values")
+    nfinset: [String] @doc(description: "Not find in set. The value can contain a set of comma-separated values")
```

Comment:
Please, cover this case with API-functional tests
We can apply this filter for product
Thanks

Reply:
@naydav I added API-functional tests

Reply:
@ronak2ram 
I mean create need to create one more testcase in ``\Magento\GraphQl\Catalog\ProductViewTest`
1) preconditions: some fixture with products (can used existed)
2) perform query with `nfinset` condition
3) check that result is proper

Thanks